api/CloseEvent.json

```diff
@@ -0,0 +1,101 @@
+{
+  "api": {
+    "CloseEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CloseEvent",
+        "support": {
+          "webview_android": {
+            "version_added": null
```

Comment:
All 3 versions of Chrome are `true`. You can find the complete list here: https://cs.chromium.org/chromium/src/third_party/blink/renderer/modules/websockets/close_event.idl

Reply:
Fixed in c2d8949f0be1c60887c66b19230011ffdac21ffa.